blog/content/post/2020-04-30-ipxe-https.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Enabling HTTPS support in iPXE
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
```

Comment:
@miczyg1 do not forget about abstract and cover image

Reply:
Added